docs/tutorials/image_classification/beginner.md

```diff
@@ -18,42 +18,21 @@ For demonstration purposes, we use a subset of the [Shopee-IET dataset](https://
 Each image in this data depicts a clothing item and the corresponding label specifies its clothing category.
 Our subset of the data contains the following possible labels: `BabyPants`, `BabyShirt`, `womencasualshoes`, `womenchiffontop`.
 
-We download the data subset and unzip it using the following commands:
+We can load a dataset by downloading a url data automatically:
 
 ```{.python .input}
-filename = ag.download('https://autogluon.s3.amazonaws.com/datasets/shopee-iet.zip')
-ag.unzip(filename)
-```
-
-After the dataset is downloaded, we load it into a [Dataset](/api/autogluon.task.html#autogluon.vision.ImageClassification.Dataset) object: 
-
-```{.python .input}
-dataset = task.Dataset('data/train')
-```
-
-Load the test dataset as follows:
-
-```{.python .input}
-test_dataset = task.Dataset('data/test', train=False)
-```
-
-If you don't have a GPU, change the dataset to 'FashionMNIST' to ensure that it doesn't take too long to run:
-
-```{.python .input}
-if ag.get_gpu_count() == 0:
-    dataset = task.Dataset(name='FashionMNIST')
-    test_dataset = task.Dataset(name='FashionMNIST', train=False)
+train_dataset, _, test_dataset = Task.Dataset.from_folders('https://autogluon.s3.amazonaws.com/datasets/shopee-iet.zip')
+print(train_dataset)
 ```
 
 ## Use AutoGluon to Fit Models
 
 Now, we fit a classifier using AutoGluon as follows:
 
 ```{.python .input}
-classifier = task.fit(dataset,
-                      epochs=5,
-                      ngpus_per_trial=1,
-                      verbose=False)
+task = Task()
+# since the original dataset does not provide validation split, the `fit` function splits it randomly with 90/10 ratio
+classifier = task.fit(train_dataset, epochs=2)  # you can trust the default config, we reduce the # epoch to save some build time
```

Comment:
there's no specific reason why it can't, the I can use `classifier = xxx()` and `classifier.fit()` without returned value. Updated for object detection as well